src/Microsoft.CodeAnalysis.Razor/ViewComponentTagHelperDescriptorProvider.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Razor.Language;
+
+namespace Microsoft.CodeAnalysis.Razor
+{
+    public sealed class ViewComponentTagHelperDescriptorProvider : RazorEngineFeatureBase, ITagHelperDescriptorProvider
```

Comment:
No not yet. We need to do multitargeting first. This has to be hardcoded, and I don't want to introduce an assembly dependency that we will undo.